CliParsing.py

```diff
@@ -1,4 +1,5 @@
 import argparse
+import sys
```

Comment:
This file contains unused source code.

*PyUnusedCodeBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/CliParsing.py
+++ b/CliParsing.py
@@ -1,5 +1,4 @@
 import argparse
-import sys
 
 
 def default_arg_parser():
```